spring/boot-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -196,6 +215,70 @@ ArmeriaServerConfigurator actuatorServerConfigurator(
         };
     }
 
+    @Bean
+    @ConditionalOnProperty("management.server.port")
+    ArmeriaServerConfigurator secureActuatorServerConfigurator(WebEndpointProperties properties,
+                                                               ManagementServerProperties serverProperties,
+                                                               ConfigurableEnvironment environment,
+                                                               ArmeriaSettings armeriaSettings) {
+        return sb -> {
+            final Port port = obtainManagementServerPort(serverProperties.getPort());
+            if (port != null) {
+                configurePorts(sb, ImmutableList.of(port));
+                addLocalManagementPortPropertyAlias(environment, port);
+                configureSecureDecorator(sb, port, properties.getBasePath(), armeriaSettings);
+            }
+        };
+    }
+
+    @Nullable
+    private static Port obtainManagementServerPort(@Nullable Integer port) {
+        return Optional.ofNullable(port)
+                       .filter(it -> it.equals(0))
```

Comment:
What happens if the management port number is not `0`?

Reply:
Oops... I fixed this. 😨